server/sonar-web/src/main/js/apps/issues/components/AppContainer.js

```diff
@@ -61,7 +61,8 @@ const fetchIssues = (query: RawQuery) => dispatch =>
     .then(response => dispatch(fetchIssueOrganizations(response.issues)).then(() => response))
     .catch(onFail(dispatch));
 
-const onRequestFail = (error: Error) => (dispatch: Dispatch<*>) => onFail(dispatch)(error);
+const onRequestFail = (error /*: Error */) => (dispatch /*: Dispatch<*> */) =>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)